BenchmarkVCFs/FindSamplesAndBenchmark.wdl

```diff
@@ -2,86 +2,86 @@ version 1.0
 import "BenchmarkVCFs.wdl" as Benchmark
 
 workflow FindSamplesAndBenchmark {
-    
+
     input {
         Array[File] input_callset
         Array[String] input_callset_labels
         Array[File] ground_truth_files
         Array[File] ground_truth_indexes
         Array[File] ground_truth_intervals
         Array[String] truth_labels
-        Array[File] annotation_intervals
 
         File? gatkJarForAnnotation
-        Array[String]? annotationNames=[]
-  
+        Array[String]? annotationNames = []
+
         File ref_fasta
         File ref_fasta_index
         File ref_fasta_dict
-        File ref_fasta_sdf
 
-        File haplotype_database 
-        File picard_cloud_jar
+        File haplotype_database = "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.haplotype_database.txt"
```

Comment:
I would say either default all reference files to hg38 or none of them, in case users want to run it in hg19

Reply:
Good point. I added hg38 for the default reference.